test/e2e/docs/config_values.md

```diff
@@ -41,10 +45,15 @@ A full list of config values are:
 | testUserName               | This is an existing test WordPress.com account for testing purposes - this account should have a **single** site                                                                                                                                         | testuser123                                                                                                                                                                 | Yes                             | **NO**         |
 | testUserNameJetpack        | This is an existing test WordPress.com account for testing purposes **that has a single jetpack site**                                                                                                                                                   | testuser123                                                                                                                                                                 | Yes                             | **NO**         |
 | testUserNameMultiSite      | This is an existing test WordPress.com account for testing purposes **that has multiple sites**                                                                                                                                                          | testuser123                                                                                                                                                                 | Yes                             | **NO**         |
+| viewportName               | Default viewport name to be used unless overriden.                                                                                                                                                                                                       | desktop                                                                                                                                                                     | Yes                             | Yes            |
+| viewportSize               | Viewport size specified as width and height in pixels for each viewportName that is suppoted.                                                                                                                                                            | desktop                                                                                                                                                                     | No                              | Yes            |
```

Comment:
```suggestion
| viewportSize               | Viewport size specified as width and height in pixels for each viewportName that is supported.                                                                                                                                                            | desktop                                                                                                                                                                     | No                              | Yes            |
```

Reply:
You sure have a keen eye...thanks for pointing these out!